README.md

```diff
@@ -24,16 +29,16 @@ for Debian](https://cfengine.com/product/community/) and that you'll run
 these commands as the root user.
 
 1. copy/clone the contents of this repository
-2. fully disable the cfengine3 service, built automatically by systemd from
-the init.d file:<br/>```systemctl mask cfengine3.service```
+2. stop the running CFEngine services, e.g. via ```/etc/init.d/cfengine3 stop```
 3. copy the unit files from the repository into `/etc/systemd/system`:<br/>```cp *.service /etc/systemd/system```<br/>(but see the note below)
 4. reload the configuration of systemd via ```systemctl daemon-reload```
-5. stop the running CFEngine services, e.g. via ```/etc/init.d/cfengine3 stop```
-6. start the new separate services:<br/> ```systemctl start cf-serverd.service```<br/>and same for cf-execd.service and cf-monitord.service
+5. enable individual services that should be started by cfengine service:<br/>```systemctl enable cf-execd.service```<br/> and same for cf-execd.service and cf-monitord.service
```

Comment:
Small mistake here: cf-execd is mentioned twice, while cf-serverd is not mentioned.